src/test/scala/io/iohk/ethereum/network/handshaker/EtcHandshakerSpec.scala

```diff
@@ -172,43 +172,43 @@ class EtcHandshakerSpec extends FlatSpec with Matchers  {
       override val appStateStorage: AppStateStorage = TestSetup.this.storagesInstance.storages.appStateStorage
     }
 
-    override lazy val blockchainConfig = new BlockchainConfig {
+    override lazy val blockchainConfig = BlockchainConfig (
```

Comment:
Here you could use the copy given `EphemBlockchainTestSetup` has access to blockchainConfig "default".